static/app/utils/events.tsx

```diff
@@ -115,9 +119,17 @@ export function getTitle(event: Event | BaseGroup, features: string[] = []) {
         };
       }
 
+      if (displayTitleWithTreeLabel) {
+        return {
+          subtitle: culprit,
+          ...computeTitleWithTreeLabel(metadata, features),
```

Comment:
We have another check for a featureflag in this function, can you consolidate all checks in there or move those checks outside of this function?